### PR TITLE
Add flagging option in assessments

### PR DESCRIPTION
**Pull request by amalieeh**
The user can flag answers. If reassessed, the flag follows and is the same as the oj assessment. It is not
used to anything particular as of now.

### CODE REVIEW

File: pages/assessment.tsx [hunk 1]
```diff
@@ -93,7 +93,7 @@ const Assessment: NextPage = () => {
     const maxReAssessmentPercentage = 0.2;
     if (
       // not currently assessing a reAssessment
-      currentPage * (maxItemsPerPage - 1) <
+      currentPage *
```
Comment (sylvihuynh) [inline pages/assessment.tsx:96]: this is always true

File: components/textbox.tsx [hunk 2]
```diff
@@ -3,21 +3,31 @@ import styles from "../styles/Textbox.module.css";
...
 const Textbox: React.FC<textboxprop> = (props: textboxprop) => {
   return (
     <div className={styles.card}>
       <div className={styles.alignItems}>
         Besvarelse fra kandidat: {props.assessment.candidateId}
-        <Pointsbox
-          assessment={props.assessment}
-          setAssessment={props.setAssessment}
-        />
+        <div style={{display: "flex"}}>
```
Comment (sylvihuynh) [inline components/textbox.tsx:20]: Since we have css file for textbox, inline styling should be a nono

File: pages/assessment.tsx (hunk 1 above)
Comment (amalieeh) [reply pages/assessment.tsx:96]: The power of code review! Nice catch! Was just a weird mistake I think

**Review by sylvihuynh — APPROVED** [review]

**Comment by sylvihuynh** [comment]
Woopwoop this issue went fast by💨 You did good!

**Pull request merged**